hack/release/build/cross.sh

```diff
@@ -48,3 +48,9 @@ export GOOS=linux GOARCH=arm64
 export GOOS=linux GOARCH=ppc64le
 EOF
 )
+
+# add sha256 for binaries
+cd "${REPO_ROOT}"/bin
```

Comment:
TODO: refactor to not need `cd`
TODO: update loop to ignore files with .sha256sum extension.